build.cake

```diff
@@ -58,28 +58,29 @@ Task("Clean")
     .Description("Cleans directories.")
     .Does(() =>
     {
-        CleanDirectory(PROJECT_DIR + "bin/");
-        CleanDirectory(PROJECT_DIR + "packages/");
+        CleanDirectory(BIN_DIR);
+        CleanDirectory(PACKAGE_DIR);
```

Comment:
Where these wrong before. `BIN_DIR` is defined as `PROJECT_DIR + "bin/" + Configuration + "/"`
and `PACKAGE_DIR` is `Argument("artifact-dir", PROJECT_DIR + "package") + "/"`, so we are cleaning different folders now

Reply:
Yes. I was comparing the Debug and Release builds using both command-line and the IDE and I was surprised to see that building Release in the command-line also cleaned my Debug folder. That seemed unexpected so I fixed it without creating a new issue - which in theory would have been the right thing to do. :-(